### PR TITLE
fix command line arguments for alterware-launcher.exe

### PR DESCRIPTION
**Pull request by mxve**
bif gix

### CODE REVIEW

**Review by diamante0018 — APPROVED** [review]
We tested this locally and this fixes an issue with the alterware launcher. please merge

**Comment by mxve** [comment]
probably still requires some work, don't merge yet

**Review by diamante0018 — APPROVED** [review]
Very good

**Review by Rackover — COMMENTED** [review]
It is not immediatly obvious what this fixes, can I have more info please? 😂 
Was this tested on a wide string locale system, like Russian, to make sure there's no regression?  (Since we're touching environment variables and system path name)

**Comment by diamante0018** [comment]
This PR was done to fix IW4x auto restarting via the alterware launcher post self-update.
It was tested on mxve's PC and mine. Yes this will absolutely not break anything on russian PC because we are using widestrings where possible which can fit non utc-8 chars like russian or japanese.

File: src/Utils/Library.cpp [hunk 1]
```diff
@@ -102,22 +102,22 @@ namespace Utils
...
+			nullptr, false, NULL, nullptr, currentDir.wstring().data(),
+			&startup_info, &process_info);
 
-		if (processInfo.hThread && processInfo.hThread != INVALID_HANDLE_VALUE)
-			CloseHandle(processInfo.hThread);
-		if (processInfo.hProcess && processInfo.hProcess != INVALID_HANDLE_VALUE)
-			CloseHandle(processInfo.hProcess);
+		if (process_info.hThread && process_info.hThread != INVALID_HANDLE_VALUE)
+			CloseHandle(process_info.hThread);
+		if (process_info.hProcess && process_info.hProcess != INVALID_HANDLE_VALUE)
```
Comment (Rackover) [inline src/Utils/Library.cpp:120]: It should be camelCase not snake_case, in IW4x that's what is used

**Review by Rackover — APPROVED** [review]
Looks good just a little formatting issue. I will merge as soon as it is addressed, and i will tag a new release

**Pull request merged**